blog-page.html

```diff
@@ -0,0 +1,285 @@
+      font-size: 3em;
+      text-align: center;
+      background-color: rgba(0, 0, 0, 0.5); /* Add a semi-transparent background */
+      padding: 20px;
+      border-radius: 10px;
+    }
+
+    /* Style for logo */
+    .logo {
+      width: 150px; /* Adjust the width as needed */
+      height: auto; /* Maintain aspect ratio */
+      display: block; /* Ensure the logo behaves as a block element */
+    }
+
+    /* Style for blog entry */
+    .blog-entry {
+      display: flex;
+      gap: 20px;
+      align-items: center;
+    }
+
+    /* Style for blog entry content */
+    .blog-entry-content {
+      flex: 1;
+    }
+
+    /* Style for popular posts section */
+    .popular-posts {
+      justify-content: end;
+      top: 150px; /* Adjust as needed */
+      right: calc(50% + 250px); /* Adjust as needed */
+      width: 300px; /* Adjust width as needed */
+      height: 500px;
+      background-color: #fff;
+      border-radius: 10px;
+      padding: 20px;
+      box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+    }
+
+    .popular-posts h4 {
+      font-size: 1.5rem;
+      margin-bottom: 1rem;
+    }
+
+    .popular-posts ul {
+      list-style: none;
+      padding: 0;
+      margin: 0;
+    }
+
+    .popular-posts img {
+      width: 50px;
+      height: 50px;
+      border-radius: 50%;
+      margin-right: 1rem;
+    }
+
+    .popular-posts .text-lg {
+      font-size: 1.2rem;
+    }
+
+    /* Center the blog entries */
+    .blog-entries-container {
+      display: flex;
+      justify-content: start;
+      margin-top: 100px; /* Adjust as needed */
+      margin-left: 5%;
+    }
+
+    .blog-entries {
+      max-width: 75%; /* Adjust as needed */
+    }
+  </style>
+</head>
+<body class="font-sans">
+
+  <header>
+    <nav class="flex items-center justify-between py-2 px-4">
+      <div>
+        <a href="/index.html">
+          <img
+            src="/assets/logo/travelup.svg"
+            alt="TravelUp logo"
+            class="logo"
+          />
+        </a>
+      </div>
+      <div class="flex space-x-4">
+        <a
+          href="/index.html"
+          class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+          >Home</a
+        >
+        <a
+          href="/about.html"
+          class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+          >About</a
+        >
+        <a
+          href="/shop.html"
+          class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+          >Shop</a
+        >
+        <a
+          href="/blog.html"
+          class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+          >Blog</a
+        >
+        <a
+          href="/contact.html"
+          class="text-travel-orange text-lg font-semibold hover:text-orange-600"
+          >Contact</a
+        >
+      </div>
+    </nav>
+    <img
+      src="/assets/images/tiago-rosado-cMG5qjpnsyg-unsplash.jpg"
+      alt="Hero image"
+      class="hero-image"
+    />
```

Comment:
set a max height so it doesn't overflow the screen, other pages will have something similar which you can transfer from